saleor/settings.py

```diff
@@ -21,6 +21,8 @@
 
 from . import patched_print_object
 
+PATTERNS_IGNORED_IN_QUERY_CAPTURES = ""
```

Comment:
Where do you sue it?

Reply:
My bad... again...

Reply:
Why did you mark it as resolved? Comments like that automatically resolves themselves after line gets changed. Do not resolve them manually unless you're sure you've changed related part and the code is pushed.